Bitkit/Services/LightningService.swift

```diff
@@ -234,6 +241,10 @@ class LightningService {
             // try? self.setMaxDustHtlcExposureForCurrentChannels()
         }
         Logger.info("LDK synced")
+
+        // Emit state change with sync timestamp
+        let syncTimestamp = UInt64(Date().timeIntervalSince1970)
+        syncStatusChangedSubject.send(syncTimestamp)
```

Comment:
This ain't bad but it would be better to read the value from `node.status()`, ie `NodeStatus.latestLightningWalletSyncTimestamp` (which is in SECONDS! not millis, so it needs to be multiplied)